server/entities/School.ts

```diff
@@ -0,0 +1,29 @@
+import { AfterLoad, Column, Entity, JoinTable, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+
+import { User } from './User';
+
+@Entity()
+export class School {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @ManyToMany(type => User)
+    @JoinTable({ name: 'students' })
+    students?: User[];
+
+    @Column('varchar', { nullable: true })
+    name: string | null;
+
+    @Column('text', { nullable: true })
+    _validationData: string | null;
```

Comment:
https://github.com/typeorm/typeorm/blob/master/docs/entities.md#column-options

Pass a transformer to @Column, instead of doing this stuff